houston/runner.py

```diff
@@ -54,7 +85,10 @@ def __init__(self, system, size, source, callback):
         self._lock = threading.Lock()
 
         # provision desired number of runners
-        self.__runners = [MissionRunner(self) for _ in range(size)]
+        if with_coverage:
+            self.__runners = [MissionRunnerWithCoverage(self, files_to_instrument) for _ in range(size)]
+        else:
+            self.__runners = [MissionRunner(self) for _ in range(size)]
```

Comment:
It looks like this code won't clean up correctly if a SIGINT or SIGTERM is received.

Reply:
@ChrisTimperley This is again not the lates version but it's still an issue in the latest version. I don't think we need to deal with this right now and we can postpone fixing it to a later time. But if you think it's important I can put some time to fix it today.

Reply:
I assumed that when I clicked "View Changes" it would show me the changes since the comment, but it appears to go back to an older version of the file.